pkg/controller/controlplane/valuesprovider.go

```diff
@@ -725,14 +729,17 @@ func (vp *valuesProvider) getControlPlaneShootChartValues(ctx context.Context, m
 		clusterAuditValues["enabled"] = true
 	}
 
-	// get apiserver ip adresses from external dns entry
-	dns := &dnsv1alpha1.DNSEntry{}
+	apiserverIPs := []string{}
+	if !extensionscontroller.IsHibernated(cluster) {
+		// get apiserver ip adresses from external dns entry
+		dns := &dnsv1alpha1.DNSEntry{}
```

Comment:
Side-note: This will not work anymore with the `UseDNSRecords` feature gate, see https://github.com/gardener/gardener/blob/master/docs/extensions/dnsrecord.md

Reply:
That's good to know. We will change this code as soon as the feature graduates.